kie-archetypes/kie-drools-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -20,25 +19,13 @@
     <junit-version>${junitVersion}</junit-version>
   </properties>
 
-  <dependencyManagement>
-    <dependencies>
-      <dependency>
-        <groupId>org.drools</groupId>
-        <artifactId>drools-bom</artifactId>
-        <type>pom</type>
-        <version>${drools-version}</version>
-        <scope>import</scope>
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
```

Comment:
Why are you removing this? I think we suggest our user to import the whole bom instead of manually specify the version to prevent mixed versions

Reply:
do we? TBH I think it's clearer this way - I think if we have an aggregator module such as `drools-classic-engine` we don't need BOMs

Reply:
Well there are many optional dependencies: decision table, persistence or even other engine like DMN. They all should use the same version so I think it makes sense to preserve the bom. Wdy?

Reply:
ok let me put it back